ext/net/02_tls.js

```diff
@@ -151,13 +151,23 @@ function listenTls({
       new Error().stack,
       "Pass the key file contents to the `Deno.ListenTlsOptions.key` option instead.",
     );
+    if (internals.future) {
+      throw new TypeError(
+        "`Deno.ListenTlsOptions.keyFile` has been removed in favor of `Deno.ListenTlsOptions.key`",
+      );
+    }
```

Comment:
Maybe instead of throwing an error we just make it `undefined` so it doesn't work (as would happen in Deno 2)

Reply:
I remember seeing some unsupported option throwing when used somewhere in the Deno codebase. I can't find where that was, so, yep, I will revert.